README.md

```diff
@@ -1,7 +1,8 @@
-# curriculum-databases-projects-template
+# VET CLINIC DATABASE 
```

Comment:
Great work adding a readme file for your project.

- [ ] It will be great to replace the old readme file with the new one provided by Microverse. This will make your project even more professional. Here is the [link](https://github.com/microverseinc/readme-template) to the new readme template

Reply:
resolved